### PR TITLE
Added message color

### PR DESCRIPTION
**Pull request by lodenrogue**

### CODE REVIEW

**Comment by lodenrogue** [comment]
Let me know if this works. First time looking through the code so I'm sure I missed something.

File: chat.html [hunk 1]
```diff
@@ -408,7 +415,7 @@
 
             let el_badges = createElement('span', {'class': 'msg-badges'});
             let el_pronoun = createElement('span', {'class': 'msg-pronoun' }, pronoun);
-            let el_message = createElement('span', {'class': 'msg-text'});
+            let el_message = createElement('span', {'class': 'msg-text', 'style': `color: ${message_color}`});
```
Comment (izzy) [inline chat.html:418]: This would produce `style="color: null"` if the color isn't set. Ideally this would have a conditional and not set anything.

**Comment by izzy** [comment]
Thanks, this looks good. I commented on a tiny thing that isn't ideal, should be an easy fix tho. :)

**Comment by lodenrogue** [comment]
Updated

**Review by izzy — APPROVED** [review]

**Comment by izzy** [comment]
Awesome work! Thanks! :)

**Pull request merged**